pkg/controller/buildrun/buildrun_controller.go

```diff
@@ -177,45 +201,27 @@ func (r *ReconcileBuildRun) Reconcile(request reconcile.Request) (reconcile.Resu
 
 	ctxlog.Debug(ctx, "start reconciling BuildRun", namespace, request.Namespace, name, request.Name)
```

Comment:
I think we now need a comment here stating that `request.Name` can be a TaskRun name or a BuildRun name.

Reply:
yes, also fixed already